doc/classes/RigidBody2D.xml

```diff
@@ -176,6 +176,28 @@
 		<member name="inertia" type="float" setter="set_inertia" getter="get_inertia" default="0.0">
 			The body's moment of inertia. This is like mass, but for rotation: it determines how much torque it takes to rotate the body. The moment of inertia is usually computed automatically from the mass and the shapes, but this property allows you to set a custom value.
 			If set to [code]0[/code], inertia is automatically computed (default value).
+			[b]Note:[/b] This value does not change when inertia is automatically computed. Use [PhysicsServer2D] to get the computed inertia.
+			[codeblocks]
+			[gdscript]
+			@onready var ball = $Ball
+
+			func get_ball_inertia():
+			    return 1 / PhysicsServer2D.body_get_direct_state(ball).inverse_inertia
+			[/gdscript]
+			[csharp]
+			private RigidBody2D ball;
+
+			public override void _Ready()
+			{
+			    ball = GetNode&lt;RigidBody2D&gt;("Ball");
+			}
+
+			private float GetBallInertia()
+			{
+			    return 1 / PhysicsServer2D.BodyGetDirectState(ball.GetRid()).InverseInertia;
+			}
```

Comment:
To follow our style guide, prefix private variables with an underscore (`_`).

```suggestion
			private RigidBody2D _ball;

			public override void _Ready()
			{
			    _ball = GetNode&lt;RigidBody2D&gt;("Ball");
			}

			private float GetBallInertia()
			{
			    return 1 / PhysicsServer2D.BodyGetDirectState(_ball.GetRid()).InverseInertia;
			}
```

Reply:
Thank you. Should be fixed now.